### PR TITLE
update schedule

### PR DESCRIPTION
**Pull request by SylviaBABY**
**Motivation**

- These docs need to transfer in Q4.
- Update the name of the task translation.

**Modifications**

- Increase docs of 「get started」*2、「Schema」*4、「Function」*7、「IO」*8.

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
@Anonymitaet could you please help review this PR? Thank you.

**Comment by Jennifer88huang** [comment]
@SylviaBABY You can learn the description content from https://github.com/apache/pulsar/pull/5512
Please keep only necessary info in the description.

File: schedule.md [hunk 1]
```diff
@@ -41,3 +41,25 @@ io-redis | Bu Xing | | In Progress
...
+functions-deploy | | | To Do
+functions-cli | | | To Do
+io-overview | | | To Do
+io-quickstart（对应 get started） | | | To Do
+io-use | | | To Do
+io-debug | | | To Do
+io-connectors | | | To Do
+io-cdc | | | To Do
+io-develop | | | To Do
+io-cli | | | To Do
```
Comment (Anonymitaet) [inline schedule.md]: Could you please check the master branch and add the following files (except ones with strikethrough)?

image

**Comment by SylviaBABY** [comment]
@Anonymitaet update,pls check

**Pull request merged**